src/xrGame/PHCommander.cpp

```diff
@@ -17,10 +17,10 @@ CPHCall::~CPHCall()
     xr_delete(m_action);
     xr_delete(m_condition);
 }
-bool CPHCall::obsolete() { return !m_action || m_action->obsolete() || !m_condition || m_condition->obsolete(); }
+bool CPHCall::obsolete() { return m_action->obsolete() || m_condition->obsolete(); }
 void CPHCall::check()
 {
-    if (m_condition && m_condition->is_true() && m_action)
+    if (m_condition->is_true() && m_action)
```

Comment:
Also here you remove check for `m_condition` but the one for `m_action` still exists